### ISSUE TITLE
add capability for fasta input SE adapter removal

### ISSUE THREAD

**Issue by msettles** [ibest/HTStream#123]
SE adapter removal currently only accept single sequence, add capability to search for multiple adapters by providing fasta sequence file name.

**Comment by samhunter** [comment]
Would it be useful to collect statistics on a per-adapter basis for this?
E.g.
adapter1 100 reads trimmed
adapter2 500 reads trimmed
etc...

**Comment by msettles** [comment]
what do you mean? really any library should only have 1 'adapter

right now, only accepts 1

**Comment by samhunter** [comment]
Yeah, that was what we thought as well... so it wasn't clear why a fasta file would be useful vs just putting something in on the command line (and maybe supporting Nextera + TruSeq Illumina adapters by default).

**Comment by msettles** [comment]
FYI, have some code in primer app to that I will then port to adapters and possible even seqscreener that will allow multiple seqs and recording of each
